cmd/print.go

```diff
@@ -59,6 +61,9 @@ type PrintRunner struct {
 
 	// providers indicates which providers are used to execute convert action.
 	providers []string
+
+	// Provider specific conf. Value assigned via provider specific flags --<provider>-<conf>.
+	providerSpecificConf map[string]*string
```

Comment:
same comment, I think there is already `conf` field in the Provider struct, maybe its better to call it providerSpecificFlags ?